config/query-builder.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+
```

Comment:
Add a few sentences here that explain what this option does

Reply:
Added, by need help with finding more words 😄